src/makeInfinityAdder.js

```diff
@@ -7,6 +7,21 @@
 
 function makeInfinityAdder() {
   // write code here
+  let sum = 0;
+
+  return function adder(num) {
```

Comment:
it's a common practice to return an anonymous / array function
since declaring with `function` special word hoists the function - you don't need it
```suggestion
  return (num = 0) => {
```